app/controllers/application_controller.rb

```diff
@@ -14,4 +14,8 @@ def user_not_authorized
     flash[:alert] = "You are not authorized to perform this action."
     redirect_to(request.referrer || root_path)
   end  
+
+  def after_sign_out_path_for(resource_or_scope)
```

Comment:
do we need this method? it just returns the root_path so couldn't we just return the root_path instead of making a new method?